content/docs/2.9/scalers/nats-jetstream.md

```diff
@@ -32,7 +32,7 @@ triggers:
 - `consumer` - Name of the consumer for a given stream.
 - `lagThreshold` - Average target value to trigger scaling actions.
 - `activationLagThreshold` - Target value for activating the scaler. Learn more about activation [here](./../concepts/scaling-deployments.md#activating-and-scaling-thresholds).(Default: `0`, Optional)
-- `useHttps` - Specifies if the NATS server monitoring endpoint is using HTTPS. Default value is false.
+- `useHTTPS` - Specifies if the NATS server monitoring endpoint is using HTTPS. Default value is false.
```

Comment:
[Based on the style guide](https://github.com/kedacore/keda-docs#writing-documentation-for-a-scaler), this should be like
```suggestion
- `useHTTPS` - Specifies if the NATS server monitoring endpoint is using HTTPS. (Default: `false`, Optional)
```